Input/InputDecoratorIcon.js

```diff
@@ -43,11 +45,9 @@ const InputDecoratorIconBase = kind({
 		 * @default 'large'
 		 * @public
 		 */
-		size: PropTypes.oneOf(['small', 'large'])
-	},
+		size: PropTypes.oneOf(['small', 'large']),
```

Comment:
Could you update the description for `size` because the default value of the `size` is  a `small` for `silicon` skin and is a `large` for other skins? `* @default 'large'` looks fine to me.

Reply:
According to my comment, the default value for all skins will be `large`. So please ignore this comment.